src/Packetizer/PassthroughPacketizer.c

```diff
@@ -0,0 +1,104 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2022 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PassthroughPacketizer.h"
+
+#include <assert.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "routing.h"
+#include "PacketizerInternal.h"
+#include "SpacePacketInternal.h"
+
+void
+PassthroughPacketizer_init(Packetizer* const self, const enum SystemBus busId, size_t* const headerSize)
+{
+    // Unused in this implementation
+    (void)self;
+    (void)busId;
+
+    *headerSize = 0;
+
+    if(bus_to_unique_port_map[busId] == INTERFACE_INVALID_ID) {
+        // Passthrough Packetizer requires an unique interface on bus.
+        abort();
+    }
+}
+
+size_t
+PassthroughPacketizer_packetize(Packetizer* const self,
+                                const Packetizer_PacketType packetType,
+                                const enum SystemBus busId,
+                                const uint16_t source,
+                                const uint16_t destination,
+                                uint8_t* const packetPointer,
+                                const size_t dataOffset,
+                                const size_t dataSize)
+{
+    // Unused in this implementation
+    (void)self;
+    (void)packetType;
+    (void)source;
+    (void)busId;
+    (void)packetPointer;
```

Comment:
Unused?

Reply:
Unused. If you mean that is is used in assert, then remember that in release build asserts are removed, hence this is required